site/content/en/docs/admission-check-controllers/provisioning.md

```diff
@@ -14,6 +14,7 @@ The Provisioning Admission Check Controller is supported on [Kubernetes cluster-
 
 ## Parameters
```

Comment:
Add a preamble along this lines:

There are two ways to configure the parameters to be sent to a ProvisioningRequest for your Jobs. You can configure via `ProvisioningRequestConfig` to set parameters to all jobs that land(are assigned?) on it. Or you can configure it per job using annotations.